db/jambones-sql.sql

```diff
@@ -140,9 +140,12 @@ PRIMARY KEY (lcr_route_sid)
 
 CREATE TABLE password_settings
 (
+password_settings_sid CHAR(36) NOT NULL UNIQUE ,
+account_sid CHAR(36) NOT NULL,
```

Comment:
The password_settings table is designed to only ever have one row -- these are the global rules for everyone on the system.  Therefore, it would be incorrect to have an account_sid in there.  And since there is only one row, I saw no need to have a password_settings_sid either -- there is never a need to use an identifier to retrieve a row, since there should only ever be zero (on startup) or one row.

And as for the case of having zero rows, then the GET api should just return default values (min length=8, no require digits, no require pins

Reply:
ah right, I forgot to read this doc: https://docs.google.com/document/d/1jciNfWDr9HQuT0wtMIKLpuwU3n5JWhzUJNjFiwGcDt0/edit#heading=h.hpv9lyb3gydg

Fixing it now

Reply:
@davehorton just fixed the branch, please help me to review it